api-description/forms/pk/get.json

```diff
@@ -5,55 +5,66 @@
         "status": 200,
         "response_data": {
             "name": "How have you been using the site?",
-            "url": "http://localhost:8000/forms/1",
```

Comment:
Yeah, ok.  I should be able to make it a `HyperlinkedModelSerializer` when we have views/urls in place, so I'll readd the url to the description.